lib/extract/netcdf/extract_netcdf_base.jinja

```diff
@@ -218,7 +219,8 @@ contains
 
         ! Open the NetCDF file
         retval = CheckError(nf90_create(module_name//"-"//region_name//".nc", &
-                                        NF90_CLOBBER, this%ncid))
+                                        IOR(NF90_NETCDF4,NF90_CLOBBER),       &
```

Comment:
What's the NF90_NETCDF4 for?

Reply:
I've added a comment (we need netcdf 4 Format in order to use 64-bit integer in NF90_INT_64 or so).

Reply:
Great, thanks.